README.md

```diff
@@ -100,12 +100,21 @@ Anywhere the Capybara DSL methods (visit, click etc.) are available so too are t
 
 ```ruby
 screenshot_and_save_page
+
+# with custom prefix
+screenshot_and_save_page('custom_prefix')
+
+# image only, with custom prefix:
+screenshot_and_save_page('custom_prefix', true)
```

Comment:
I am afraid I cannot approve this.  I am happy to support this PR, however we need to see;

* [ ] Use of options in this method so that adding new options in future is more easily supported, and the code is more readable.  For example, I'd expect to see `screenshot_and_save_page(prefix: 'custom_prefix', html: true)`.  
* [ ] I am not sure even the former arg `html` makes sense.  What does that even mean, given your comment says "image only" and the default is true?
* [ ] I'd like to see this approach with argument `options = {}` carried through internal methods
* [ ] We need to have some tests for this, even if very basic, so that changes in future won't introduce regressions.